### PR TITLE
test: add specs for PtElementInfo

### LINKED ISSUE #40: Add test for PtElementInfo

### PR DESCRIPTION
**Pull request by rubenmoya**
Closes https://github.com/OpenSourceRaidGuild/atom-pwa/issues/40

### CODE REVIEW

File: src/components/pt-element/PtElementInfo.test.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import PtElementInfo from "./PtElementInfo";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+const IRON_ELEMENT = {
```
Comment (mpeyper) [inline src/components/pt-element/PtElementInfo.test.tsx]: Can we just import one of the `json` files instead? Would make keeping it in sync a bit easier.

Comment (rubenmoya) [reply src/components/pt-element/PtElementInfo.test.tsx]: Sure!

File: src/components/pt-element/PtElementInfo.test.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    expect(screen.getByText(IRON_ELEMENT.symbol)).toBeInTheDocument();
+  });
+
+  it("shows the elements name", () => {
+    render(<PtElementInfo element={IRON_ELEMENT} />);
+
+    expect(screen.getByText(IRON_ELEMENT.name)).toBeInTheDocument();
+  });
+
+  it("adds the element group as a classname", () => {
```
Comment (mpeyper) [inline src/components/pt-element/PtElementInfo.test.tsx:26]: This test feels a bit odd, but I don't have any better suggestions for you.

**Review by mpeyper — APPROVED** [review]

File: src/components/pt-element/PtElementInfo.test.tsx (hunk 2 above)
Comment (rubenmoya) [reply src/components/pt-element/PtElementInfo.test.tsx:26]: yeah, it feels weird, but I can't think of a better alternative besides not testing it

**Pull request merged**